package/src/components/StripeForm/v1/StripeForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  CardNumberElement,
+  CardExpiryElement,
+  CardCVCElement,
+  PostalCodeElement,
+  injectStripe
+} from "react-stripe-elements";
+import styled from "styled-components";
+import { applyTheme, withStripeElements } from "../../../utils";
+
+const Field = styled.div`
+  background-color: ${applyTheme("color_black02")};
+  padding: 10px 12px 10px 12px;
+  border-radius: 2px;
+  border-bottom: solid 1px rgba(0, 0, 0, 0.12);
+  margin-bottom: 20px;
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+`;
+
+const createOptions = () => ({
+  style: {
+    base: {
+      fontSize: applyTheme("font_size_default")(),
+      color: applyTheme("color_black55")(),
+      fontFamily: applyTheme("font_family")()
+    }
+  }
+});
+
+class StripeForm extends Component {
+  static propTypes = {
+    /**
+     * Card's CVV text placeholder
+     */
+    cardCVCPlaceholder: PropTypes.string,
+    /**
+     * Card's expiry date text placeholder
+     */
+    cardExpiryPlaceholder: PropTypes.string,
+    /**
+     * Card's Number text placeholder
+     */
+    cardNumberPlaceholder: PropTypes.string,
+    /**
+     * Card's billing postal code text placeholder
+     */
+    postalCodePlaceholder: PropTypes.string,
+    /**
+     * The stripe object which provides methods for tokenizing data and more. 
+     * See https://stripe.com/docs/stripe-js/reference#the-stripe-object for more details.
+     */
+    stripeRef: PropTypes.func
```

Comment:
Probably best to make this `.isRequired` since you call it without checking to see if it's a function (and you have a default function).